Add note about where to input the custom ui url in the policy. (#46472)
Azure/azure-content: articles/active-directory-b2c/customize-ui-overview.md

## Custom HTML and CSS

> If you wish to design your own policy layout with your customized HTML and CSS, you can do so by switching the "Use custom page content" toggle for each of the Layout names present in your policy. Please follow the below instructions regarding the custom layout configurations:

Reviewer: Hi @JoseMSoares-MSFT, this is a good call-out, but it'd be better located if you move it down under the **Get started with custom HTML and CSS** heading. Replace the first sentence in that section with what you have here, with just a couple edits.

```suggestion
If you wish to design your own policy layout with your customized HTML and CSS, do so by switching the **Use custom page content** toggle for each **Layout name** present in your policy. Then follow these guidelines for formatting and serving your HTML and CSS content.
```

![image](https://user-images.githubusercontent.com/13394541/72820356-aea82b00-3c23-11ea-9dde-b37b1093a2fd.png)

Author: Looks good to me. I've been adding these as per customers' feedback from our cases.

Reviewer: Looks like this might be related: https://github.com/MicrosoftDocs/azure-docs/issues/46392

We could probably be even more clear about selecting the different layouts under **Layout name** by adding a screenshot. I can do that on another edit pass if you'd like, or you can take a crack at it. I'm OK either way.
